orchagent/port.cpp

```diff
@@ -18,12 +20,15 @@ namespace swss {
 sai_status_t Port::bindAclTable(sai_object_id_t& group_member_oid, sai_object_id_t table_oid)
 {
     sai_status_t status;
+
     sai_object_id_t groupOid;
 
     // If port ACL table group does not exist, create one
     if (m_acl_table_group_id == 0)
     {
-        sai_object_id_t bp_list[] = { SAI_ACL_BIND_POINT_TYPE_PORT };
+        int32_t bp_list[] = {SAI_ACL_BIND_POINT_TYPE_PORT,
+                SAI_ACL_BIND_POINT_TYPE_LAG,
+                SAI_ACL_BIND_POINT_TYPE_VLAN};
```

Comment:
depending on the port type (phy/lag/vlan), we can select the BIND_POINT_TYPE here. We do not need to add all bind point type in the SAI_ACL_TABLE_GROUP_ATTR_ACL_BIND_POINT_TYPE_LIST. It is not necessary. On some platforms, it can cost us to use more resources.